wagtail/wagtailimages/tests/tests.py

```diff
@@ -81,6 +81,12 @@ def test_image_tag_with_filters(self):
         self.assertTrue('width="400"' in result)
         self.assertTrue('height="300"' in result)
 
+    def test_image_tag_with_chained_filters(self):
+        result = self.render_image_tag(self.image, 'width-400|height-150')
+        self.assertTrue('width="200"' in result)
+        self.assertTrue('height="150"' in result)
```

Comment:
This doesn't actually prove the first filter was run. How about ``fill-200x200|height-150``? The fill operation would alter the aspect ratio which should have an effect on the final width